src/app/components/ATIAnalytics/types.ts

```diff
@@ -98,10 +98,50 @@ export interface ReverbDetailsProviders {
   atiData: ATIData;
 }
 
+export type ReverbPageVars = {
+  name?: string | null;
+  additionalProperties?: {
+    // eslint-disable-next-line camelcase
+    content_language?: string | null;
+    type?: string | null;
+  };
+  destination?: string | null;
+  producer?: string | null;
+  contentId?: string | null;
+  contentType?: string | null;
+};
+
+export type ReverbUserVars = {
+  isSignedIn: boolean;
+};
+
+export type ReverbEventDetails = {
+  anchorElement?: HTMLElement;
+  experience?: {
+    // eslint-disable-next-line camelcase
+    engine_id: string;
+  };
+  event?: {
+    category: string;
+    action: 'select' | 'view';
```

Comment:
Should we create a type for this?

Reply:
I tend to not do that until its used more than once.

Reply:
Cool, makes sense 👍